src/index.js

```diff
@@ -36,7 +37,29 @@ class App extends React.Component {
       behavior: "smooth"
     });
   };
-  componentDidMount() {}
+  handleScroll = () => {
```

Comment:
handleScroll concerns with Education section and not whole App, so it should be inside Education and not App component